scripts/style.sh

```diff
@@ -55,21 +55,24 @@ case "$version" in
     ;;
 esac
 
+# Ensure that tools in `Mintfile` are installed locally to avoid permissions
+# problems that would otherwise arise from the default of installing in
+# /usr/local.
+export MINT_PATH=Mint
+
+function swiftformat() {
+  mint run swiftformat "$@"
+}
+
 system=$(uname -s)
 if [[ "$system" == "Darwin" ]]; then
   version=$(swiftformat --version)
```

Comment:
This (and the below) can probably be omitted as well, since the swiftformat version will be enforced by the Mintfile. Maybe instead they can be replaced by a check for a local Mint installation?

Reply:
I'd prefer to leave the version log in because it makes it easier to directly see in CI logs what happened. The comment about not checking the version is also useful because it contrasts with `clang-format`, which we do rigorously check.

Reply:
Actually, looking at the check failure this CL induced, the first run of `mint run swiftformat` installs it and that turns out to be this version check. It gives output like this:
```
Found: clang-format version 10.0.1 
Found: 🌱 Cloning SwiftFormat 0.45.5
🌱 Resolving package
🌱 Building package
🌱 Installed SwiftFormat 0.45.5
🌱 Running swiftformat 0.45.5...
0.45.5
```

... which is a little ridiculous. I've removed the whole section now since this will be obvious enough.